.github/workflows/node.js.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       with:
         mongodb-version: ${{ matrix.mongodb-version }}      
     - run: npm ci
-    - run: npm run ci
+    - run: npm run lint && npm run test
```

Comment:
Annoyingly `@semantic-release/npm` doesn't play nicely with multiple versions of node so I had to run both of these at the top level: https://github.com/semantic-release/npm/issues/524

In another PR, we can probably deprecate versions of node prior to 14.